controllers/operator/configmap.go

```diff
@@ -382,6 +388,20 @@ func (u *cmUpdater) CreateOrUpdate(ctx context.Context, cl client.Client, authCR
 
 func updatePlatformAuthIDP(observed, generated *corev1.ConfigMap) (updated bool, err error) {
 	updateFns := []func(*corev1.ConfigMap, *corev1.ConfigMap) bool{
+		updatesAlways(
+			"ROKS_URL",
+			"ROKS_USER_PREFIX",
+			"ROKS_ENABLED",
+			"BOOTSTRAP_USERID",
+			"CLAIMS_SUPPORTED",
+			"CLAIMS_MAP",
+			"SCOPE_CLAIM",
+			"NONCE_ENABLED",
```

Comment:
any reason this should always be set?

Reply:
Our documentation suggests that this can be updated from the CommonService CR, which should in turn update our Authentication CR: https://www.ibm.com/docs/en/cloud-paks/foundational-services/4.9?topic=options-configuring-foundational-services